doc/posts/2023/2023_05_19_Shilpi_week0_journey_to_GSoC.rst

```diff
@@ -0,0 +1,46 @@
+My Journey to GSoC 2022
+=======================
+
+.. post:: May 19 2023
+    :author: Shilpi Prasad
+    :tags: google
+    :category: gsoc
+
+
+
+About Myself
+~~~~~~~~~~~~
+
+Hey there! I'm Shilpi, a Computer Science and Engineering undergrad at Dayananda Sagar College of Engineering, Bangalore. I'm on track to grab my degree in 2024.
+My relationship with Python started just before I started college - got my hands dirty with this awesome Python Specialization course on Coursera. 
+When it comes to what makes me tick, it's all things tech. I mean, new technology always excites me. Ubuntu, with its fancy terminal and all, used to intimidate me at first, but now, I get a thrill out of using it to do even the simplest things.
+Up until 2nd year I used to do competitive programming and a bit of ML.But from 3rd year I've been into ML very seriously, doing several courses on ML as well solving ML problems on kaggle. ML is very fun and I've done a few project on ML as well. 
+Coding? Absolutely love it. It's like, this is what I was meant to do, y'know? I got introduced to git and GitHub in my first year - was super curious about how the whole version control thing worked. And then, I stumbled upon the world of open source in my second year and made my first contribution to Tardis.
+Initially, I intended on doing GSoC during my second year but ended up stepping back for reasons. This time, though, I was fired up to send in a proposal to at least one organization in GSoC. And, well, here we are!
+
+
+Intro to Open-Source and GSoC
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+So, I started off finding out about GSoC - how many hours do selected folks put in, the kind of projects people usually tackle, and all that stuff. To get a handle on what they want in a proposal, I turned to some successful ones from previous years. Really gave me an idea of the kind of stuff they expect you to bring to the table.
+Trying to find the organization that'd go with my skill set, I stumbled upon Python Software Foundation, and I was like, "This is it!". And under PSF, there was DIPY. 
+Diving into DIPY's docs was a breeze as they've got it so well put together that I managed to get my head around a completely new topic, "Diffusion MRI", just by going through their introductory docs and a bit of gpt.
+While exploring DIPY, I noticed this issue that needed a new feature. It took a good bit of reading to really understand what they were looking for and how to actually build that feature. And then, I submitted my first PR (`check it out here: https://github.com/dipy/dipy/pull/2749`)! Getting it merged wasn't exactly easy - there was a lot of room for improvement up in my code, but honestly,I feel like it's all part of the learning curve.
```

Comment:
- check it out here:   link is not working. or it is not intended to be a link ?
- `honestly,I `. missing space